docs/components/dynamic_web_content.rst

```diff
@@ -81,17 +81,17 @@ The following values are available:
 
 - **Is a translation of** - If you're creating a slot in a second language translation - for example to use on a multilingual website - select the original base language Dynamic Web Content item which you're translating. The same slot displays the appropriate language based on the Campaign or filters set, but Mautic shows the translated content if a visitor is viewing the page in a different browser language.
 
-- **Published** - Whether the Dynamic Web Content item is available for use - published - or not available - unpublished
+- **Available** - Whether the Dynamic Web Content item is available for use or not available
 
 - **Is Campaign based** - if set to Yes, Mautic pushes this Dynamic Web Content to Contacts through a Campaign. When set to No, you can specify filters for visitors to see the content.
 
 - **Requested slot name** - shown if using non-Campaign based Dynamic Web Content, this allows you to specify the slot name on your website in which the Contact sees the content.
 
 .. vale off
 
-**Publish at (date/time)** - This allows you to define the date and time at which this Dynamic Web Content item is available for displaying to Contacts
+**Enable at (date/time)** - This allows you to define the date and time at which this Dynamic Web Content item is available for displaying to Contacts
```

Comment:
```suggestion
**Enable at (date/time)** - This allows you to define the date and time at which this Dynamic Web Content item is available for displaying to Contacts
```

DWC doesn't actually have activate / deactivate date/times:

![screenshot-mautic ddev site-2024 06 13-15_17_47](https://github.com/mautic/user-documentation/assets/2930593/54b944f2-d3e1-4cc0-adfb-595bd15462b0)

Maybe this is a feature we haven't yet got merged into core? cc @escopecz ?

Reply:
I think it was just copy-pasted from other entity docs. I'm not aware of any PR that's adding it.

Reply:
It’s probably there for months 🤣